overlord/devicestate/handlers_install.go

```diff
@@ -892,8 +892,36 @@ func (m *DeviceManager) loadAndMountSystemLabelSnaps(systemLabel string) (
 	return sys, snapInfos, snapSeeds, mntPtForType, unmount, nil
 }
 
+func copySeedDir(src, dst string) error {
+	if err := os.MkdirAll(filepath.Dir(dst), 0755); err != nil {
+		return err
+	}
+	// Note that we do not use the -a option as cp returns an error if trying to
+	// preserve attributes in a fat filesystem. And this is fine for files from
+	// the seed, that do not need anything too special in that regard.
+	if output, stderr, err := osutil.RunSplitOutput("cp", "-rT", src, dst); err != nil {
+		return osutil.OutputErrCombine(output, stderr, err)
+	}
+	return nil
+}
+
+func copyLabelToSeedPartition(label, seedMntDir string) error {
+	// TODO we are copying all snaps from snaps/ folder, we should look at
+	// the assertions from the label to identify which are really needed,
+	// as more than one label might exist.
```

Comment:
it might sense to make installing/copying a seed a method of seed.Seed for the UC20+ variant exposed as other things via an interface